Assets/MirageXR/ContentTypes/Audio/Editors/Mobile/AudioCaptionEdit.cs

```diff
@@ -0,0 +1,89 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class AudioCaptionEdit : MonoBehaviour
+{
+    [SerializeField] private Button _btnBackPreview;
+    [SerializeField] private Button _btnDoneEditing;
+    [SerializeField] private TMP_InputField _captionEditText;
+
+    [SerializeField] private GameObject _panelMain;
+    [SerializeField] private GameObject _panelCaptionEdit;
+    [SerializeField] private GameObject _panelCaptionPreview;
+
+    [SerializeField] private AudioCaptionPreview audioCaptionPreview;
+    private string text = string.Empty;
+
+    // Start is called before the first frame update
+    private void Start()
```

Comment:
fix the code formatting and write more detailed error logs, not just the name of the prefab.